scripts/next-release.sh

```diff
@@ -1,10 +1,4 @@
 #!/usr/bin/env bash
-# This script implements the time-based version scheme from RFC 795
```

Comment:
Why delete this, particularly since client releases are moving to be more aligned with Sourcegraph releases, not less?

Reply:
The script does not work like the comment says. Also, we wanted "prettier" version numbers for GA.